### DISCUSSION TITLE
How to assign piggybank to transfer after init creation?

### DISCUSSION THREAD

**Discussion by nickstanley574** [firefly-iii/firefly-iii#5732] [Firefly III] [answered]
It seems you can only assign a piggybank to a transfer when it is created. You assign a piggybank post creation.  

Is is attended behavior? Is there a step or setting I am missing?

![piggybank](https://user-images.githubusercontent.com/40447503/152816171-51d50871-792a-409d-81fe-9adacc3518c7.gif)

**Comment by JC5** [comment, marked as answer]
After the transfer is created, you can no longer assign it a piggy. Instead, you can add the money to the piggy bank directly.

**Comment by skorvek** [comment]
Does this also apply to rules?  I can modify already created transactions by running an on-create rule manually, but cannot seem to get it to work for a piggy bank association. I get "success x transactions were modified" but nothing appears differently. The piggy bank is set with a start date prior to these transactions.

**Reply by JC5** [reply]
It doesn't create an event, it only updates the amount, if the transaction is a transfer. There was a bug however that this would sometimes alert you of a success while nothing happened. That will be fixed in the new release.
